### DISCUSSION TITLE
Shouldn't docs/docs/apis/proto be added to .gitignore?

### DISCUSSION THREAD

**Discussion by doncicuto** [zitadel/zitadel#5582] [Q&A] [answered]
Hi there. When building Zitadel binaries several markdown files documenting proto files are generated but those fields shouldn't be pushed to the repo and be ignored, shouldn't them?

![image](https://user-images.githubusercontent.com/30386061/229277907-e160a38e-8a41-499c-bb1b-e0e1d1180b31.png)

Current .gitignore file:

```
/internal/api/assets/authz.go
/internal/api/assets/router.go
/internal/api/ui/console/static/*
docs/docs/apis/auth
docs/docs/apis/admin
docs/docs/apis/mgmt
docs/docs/apis/system
--> docs/docs/apis/proto
```

**Comment by fforootd** [comment, marked as answer]
Hm yes, you can add them to the gitignore. There is an upcoming change where we switch the go proto build to buf as well which gets rid of this.

**Reply by doncicuto** [reply]
Thanks @fforootd
